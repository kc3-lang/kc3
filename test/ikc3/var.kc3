quote (Tag) ?
(Tag) ?
quote (U8) ?
(U8) ?
