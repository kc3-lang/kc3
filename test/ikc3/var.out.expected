?
?
(U8) ?
(U8) ?
